redis-objects.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   # Only fix this one version or else tests break
-  spec.add_dependency "redis", "~> 3.3"
+  spec.add_dependency "redis"
```

Comment:
Better to always specify a version. If the 4.x versions works then could be nice to write it this way:
`spec.add_dependency "redis", "~> 3.3", "< 5.0"`

Reply:
I did fix! thanks!